TypeDuck/Localization/zh-HK.lproj/Localizable.strings

```diff
@@ -21,27 +21,31 @@
 
 "Settings.Window.Title" = "TypeDuck";
 "SettingsView.NavigationTitle.Settings" = "設定";
-"SettingsView.NavigationTitle.Candidates" = "候選詞";
-"SettingsView.NavigationTitle.Help" = "輔助說明";
+"SettingsView.NavigationTitle.Help" = "輔助説明";
 "SettingsView.NavigationTitle.About" = "關於";
 
 "SettingsView.CandidatesView.Title" = "TypeDuck";
-"SettingsView.HelpView.Title" = "輔助說明";
+"SettingsView.HelpView.Title" = "輔助説明";
 "SettingsView.AboutView.Title" = "關於";
 
-"Emoji Suggestions" = "Emoji 候選詞";
-"Candidate Count Per Page" = "每頁候選詞數目";
+"SettingsView.CandidatesView.Settings.EmojiSuggestions" = "表情符號候選詞";
+"SettingsView.CandidatesView.Settings.CandidateCountPerPage" = "每頁候選詞數目";
 
-"Enter/Exit Options View" = "進入／退出快捷設定䈎面";
-"Directly toggle specific option" = "直接選擇某個快捷設定選項";
-"Remove highlighted Candidate from User Lexicon" = "從用戶詞庫刪除當前高亮(highlighted)候選詞";
-"Clear current pre-edit text" = "清除當前預輸入音節";
+"SettingsView.Colon" = "：";
 
-"Highlight previous Candidate" = "高亮(Highlight)上一個候選詞";
-"Highlight next Candidate" = "高亮(Highlight)下一個候選詞";
-"Backward to previous Candidate page" = "退返到上一䈎候選詞";
-"Forward to next Candidate page" = "翻去到下一䈎候選詞";
-"Jump to the first Candidate page" = "跳到第一䈎候選詞";
-"or" = "或";
+"SettingsView.HelpView.Shortcut.ToggleOptionsView" = "顯示／隱藏快捷選項頁面";
+"SettingsView.HelpView.Shortcut.ToggleOption" = "直接切換特定快捷選項";
+"SettingsView.HelpView.Shortcut.ToggleOption.NumberKeys" = "數字鍵";
+"SettingsView.HelpView.Shortcut.RemoveCandidate" = "從用戶詞庫刪除目前候選詞";
```

Comment:
The HK translation of “highlighted” would have been “反白”, but we try to avoid this word and use “目前” instead so as to stick to users’ cognitive perception.